eo-maven-plugin/pom.xml

```diff
@@ -443,6 +444,13 @@ SOFTWARE.
               </excludes>
             </configuration>
           </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <excludedGroups>""</excludedGroups>
```

Comment:
@levBagryansky Should we use `""` here? Can we leave just empty tag like `<excludedGroups/>` ?

Reply:
@volodya-lombrozo empty tag does not override <excludedGroups/> in profile but `""` does.

Reply:
@levBagryansky Ok then.

Reply:
@levBagryansky a more canonical way is to use something like `"nothing"`, but not `""`. The empty string you use looks like a bug to a reader who is not familiar with the situation, while "nothing" clearly explains what's going on. You can even use `"nothing-to-exclude"`